collection-ionic.yaml

```diff
@@ -1,9 +1,8 @@
----
```

Comment:
the yaml linter is warning about this first line of `---` being deleted, though the workflow is still passing

Reply:
Fixed in https://github.com/gazebo-tooling/gazebodistro/pull/165/commits/a979b044560e89fb6dfc5d9a471bf5a0bff5e09c